xclim/indices/_synoptic.py

```diff
@@ -1,6 +1,8 @@
 # noqa: D100
 from __future__ import annotations
 
+from typing import Tuple
```

Comment:
T|his import is probably as you can use `tuple` (lowercase) when defining types.

Reply:
You're right. The new annotations are only enabled on xclim and I forget that we can use them here. Will modify.